### DISCUSSION TITLE
Alignment math equations with numbers

### DISCUSSION THREAD

**Discussion by victorballester7** [typst/typst#3421] [Q&A]
### Description

What is the equivalent version of 
\begin{align}
...
\end{align}
in typst? I mean to put a number in each equation an keep them aligned. I would like even if I could add a number only to a few of them (like with \nonumber in latex).
Thanks

### Use Case

It would be useful to complement the equation's environment in typst.

**Comment by xkevio** [comment]
The equivalent would be just using `&` in block equations for alignment points. They work the exact same way as in LaTeX even, see: https://typst.app/docs/reference/math/#alignment. More precisely, it works like the `alignat` environment as Typst doesn't automatically add spacing, you have to do that yourself.

As for numbering, just do the usual and set a `numbering` parameter: `#set math.equation(numbering: "(1)")` for example. Currently, this will enumerate all block equations but in the future you will have more fine-grained control which equations you want to number (see #1849)
